src/containers/WorkspaceDetails.tsx

```diff
@@ -57,39 +62,91 @@ class WorkspaceDetails extends React.PureComponent<Props> {
       this.props.requestWorkspaces();
     }
     const showAlert = this.workspaceDetailsPageRef.current?.showAlert;
-    this.showAlert = (variant: AlertVariant, title: string, timeDelay?: number) => {
+    this.showAlert = (title: string, variant?: AlertVariant) => {
       if (showAlert) {
-        showAlert(variant, title, timeDelay);
+        showAlert(variant ? variant : AlertVariant.danger, title);
       } else {
         console.error(title);
       }
     };
   }
 
+  public componentDidUpdate(): void {
+    const namespace = this.props.match.params.namespace;
+    const workspaceName = this.props.match.params.workspaceName;
+
+    const workspace = this.props.allWorkspaces?.find(workspace =>
+      workspace.namespace === namespace && workspace.devfile.metadata.name === workspaceName);
+    if (!workspace) {
+      this.props.history.replace({ pathname: '/workspaces' });
+    }
+  }
+
   render() {
     return (
       <WorkspaceDetailsPage
         ref={this.workspaceDetailsPageRef}
         onSave={(workspace: che.Workspace) => this.onSave(workspace)}
+        onAction={(action => this.onAction(action))}
       />
     );
   }
 
+  async onAction(action: Actions): Promise<void> {
+    const namespace = this.props.match.params.namespace;
+    const workspaceName = this.props.match.params.workspaceName;
+    const workspace = this.props.allWorkspaces?.find(workspace =>
+      workspace.namespace === namespace && workspace.devfile.metadata.name === workspaceName);
+
+    if (!workspace) {
+      this.showAlert('Unable to find the workspace');
+      return;
+    }
+
+    switch (action) {
+      case Actions.OPEN:
+        this.props.history.replace({ pathname: `/ide/${namespace}/${workspaceName}` });
+        break;
+      case Actions.OPEN_IN_VERBOSE_MODE:
+        try {
+          await this.props.startWorkspace(workspace.id, { debug: true });
```

Comment:
I don't know if it's dashboard next or workspace client bug but debug mode does not work because of an incorrect argument 

![Screenshot_20201207_121635](https://user-images.githubusercontent.com/5887312/101338911-53584900-3886-11eb-8686-df908cd37df0.png)

![Screenshot_20201207_121657](https://user-images.githubusercontent.com/5887312/101338916-53f0df80-3886-11eb-9929-13ed18c89a1a.png)

Devfile to test
```yaml
apiVersion: 1.0.0
metadata:
  name: java-maven-5huz1
attributes:
  persistVolumes: 'false'
projects:
  - name: console-java-simple
    source:
      location: 'https://github.com/che-samples/console-java-simple.git'
      type: git
      branch: java1.11
components:
  - id: redhat/java/latest
    preferences:
      java.server.launchMode: Standard
    type: chePlugin
  - mountSources: true
    endpoints:
      - attributes:
          public: 'false'
        name: debug
        port: 5005
    command:
      - exit
    args:
      - '1'
    memoryLimit: 512Mi
    type: dockerimage
    volumes:
      - name: m2
        containerPath: /home/user/.m2
    image: 'quay.io/eclipse/che-java11-maven:7.22.0'
    alias: maven
    env:
      - value: ''
        name: MAVEN_CONFIG
      - value: '-XX:MaxRAMPercentage=50 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10 -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Dsun.zip.disableMemoryMapping=true -Xms20m -Djava.security.egd=file:/dev/./urandom -Duser.home=/home/user'
        name: MAVEN_OPTS
      - value: '-XX:MaxRAMPercentage=50 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10 -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Dsun.zip.disableMemoryMapping=true -Xms20m -Djava.security.egd=file:/dev/./urandom'
        name: JAVA_OPTS
      - value: '-XX:MaxRAMPercentage=50 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10 -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Dsun.zip.disableMemoryMapping=true -Xms20m -Djava.security.egd=file:/dev/./urandom'
        name: JAVA_TOOL_OPTIONS
commands:
  - name: maven build
    actions:
      - workdir: '${CHE_PROJECTS_ROOT}/console-java-simple'
        type: exec
        command: mvn clean install
        component: maven
  - name: maven build and run
    actions:
      - workdir: '${CHE_PROJECTS_ROOT}/console-java-simple'
        type: exec
        command: mvn clean install && java -jar ./target/*.jar
        component: maven

```

Reply:
I have fixed